joomla/joomla-cms
# New pagebreak plugin

I made this addition as requested by @mbabker in #7750 
### This PR conatin
- JHtml bootstrap sliders and tabs
- JLayout implementation for the navigation
- Code style / code cleaning
- Rename the old Mootools tabs to `Mootools tabs` and `Mootools slider`. The new tabs made with Bootstrap have the name `Tabs` and `Sliders`
### Test instructions
1. Apply the patch, and confirm that all parameters are working like they used to do. (Please notice the "Presentation Style" parameter values `Tabs` and `Sliders` are the new Bootstrap Tabs).
### Notes

I'm not sure if this plugin is allready correct and does what it has to do, also I'm not a pro developer. I simply made addition to the old plugin. So please comment on my code, give me hints and tips and tell me if I'm on the good why or not :blush:
